src/modules/main.js

```diff
@@ -0,0 +1,150 @@
+const list = document.getElementById('listtodo');
+const add = document.getElementById('add');
+
+const completed = false;
+let index = 1;
+let todoList = [];
+
+const showList = () => {
+  let litItems = '';
+  todoList.forEach((element) => {
+    if (element.completed === true) {
+      litItems += `<li class="task">
+                                  <div>
+                                  <input type="checkbox" class="checkFunc" checked onclick="checkFunc(${element.index - 1});" id="check${element.index - 1}">
```

Comment:
[REQUIRED] Kindly make this a change event listener instead of click

Reply:
@fobadara please can you clarify i didn't understand what you mean